schemas/constructs/v1beta1/academy/openapi.yml

```diff
@@ -172,6 +172,13 @@ components:
       enum:
         - learning-path
         - challenge
+        
+    Level:
+      type: string
+      enum:
+        - introductory
```

Comment:
beginner or introductory? I'm seeing a discrepancy.

Reply:
oh yeah , i forgot about this pr and ended up doing this in one of other merged prs